### ISSUE TITLE
cowpeamine genotype matrix displayer busted

### ISSUE THREAD

**Issue by adf-ncgr** [legumeinfo/mine-issues#62]
e.g. https://mines.legumeinfo.org/cowpeamine/genotypingstudy:CB27_x_IT82E-18.gen.Pottorff_Li_2014
console reports:
report.do?id=90000002:1008 Uncaught Error: Failed to load Displayer "GenotypeMatrixDisplayer" error
    at Object.error (report.do?id=90000002:1008:20)
    at l (jquery.min.js:4:24882)
    at Object.fireWith [as rejectWith] (jquery.min.js:4:25702)
    at k (jquery.min.js:6:5373)
    at XMLHttpRequest.<anonymous> (jquery.min.js:6:9152)
report.do?id=90000002:1581 

Although we may be revisiting the use of intermine for genotype data, it would be good to have this back for datasets on which it is feasible until we have an alternative.

**Comment by sammyjava** [comment]
OH, yeah I'm still supporting the mstmap loads (which are Riverside-specific? I think they wrote it). Those are much lower density then the massive VCFs elsewhere, the biggest is the MAGIC-2017 file, 30 MB uncompressed. I'll fix it, not sure why it broke, probably some upstream data model change in 5.0.6.1, which is what CowpeaMine is on.

**Comment by sammyjava** [comment]
Turns out this is a data loading bug. The GWAS were loaded but without any GWASResults from the MSTMap loader.  I'm building a new CowpeaMine on 5.0.3.6 since it's easier than trying to squeeze the data into a production mine. I'll figure out what the deal was with lis-mstmap.

**Comment by sammyjava** [comment]
OK it's bigger than that, it's the merge-markers-on-name thing that I've dealt with in other cases. The genetic data do not specify _which_ genome the markers belong to and I've made updates to allow a marker to be mapped to several genomes. But I hadn't updated the GenotypingStudy stuff accordingly. So we've got dupe markers, one from the GFF and actually two from MSTMap loads that only have names. Will fix. Non-trivial.
